sql-cli/pyproject.toml

```diff
@@ -20,20 +20,29 @@ classifiers = [
     "Programming Language :: Python :: 3.8",
     "Programming Language :: Python :: 3.9",
 ]
-dependencies = [
-    "networkx",
-    "typer[all]>=0.6.1"
-]
+
 [project.urls]
 Home = "https://astronomer.io/"
 Source = "https://github.com/astronomer/astro-sdk/sql-cli"
 
-[project.optional-dependencies]
-tests = [
-    "pytest",
-    "pytest-cov",
-    "mypy"
-]
+[tool.poetry.dependencies]
+python = ">=3.8,<4.0.0"
+typer = {extras = ["all"], version = "^0.6.1"}
+networkx = "^2.8.7"
+Jinja2 = "^3.1.2"
```

Comment:
This version won't work with Airflow. For example, airflow 2.2.5 uses `jinja2>=2.10.1,<3.1` 

https://github.com/astronomer/airflow/blob/2.2.5/setup.cfg#L131

airflow 2.0.0 has even `jinja2>=2.10.1, <2.12.0`

https://github.com/astronomer/airflow/blob/2.0.0/setup.cfg#L111

Reply:
We support airflow >2.0.0 right? So we should set it in my opinion to `jinja2>=2.10.1`

Reply:
It may be worth to have an upper bound similar to Airflow, so we don't start using versions which will cause dependency conflicts to end-users